diff -up openobex-1.7.1-Source/apps/obex_tcp.c.netdb openobex-1.7.1-Source/apps/obex_tcp.c
--- openobex-1.7.1-Source/apps/obex_tcp.c.netdb	2012-03-28 21:18:23.000000000 +0200
+++ openobex-1.7.1-Source/apps/obex_tcp.c	2015-08-28 12:41:17.985028548 +0200
@@ -22,7 +22,7 @@
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
-#define _XOPEN_SOURCE 520
+#define _DEFAULT_SOURCE
 
 #ifdef _WIN32
 #include <winsock2.h>
